src/ws/cockpitwebservice.c

```diff
@@ -668,7 +658,6 @@ process_transport_init (CockpitWebService *self,
       else
         {
           json_object_set_boolean_member (object, "superuser", FALSE);
-          cockpit_creds_poison (self->creds);
```

Comment:
We should still poisen the creds here for a bridge with "explicit_superuser_capability".

Reply:
Now we only add the "superuser" field to the "init" message for bridges with "explicit_superuser_capability".